modules/openapi-generator-maven-plugin/src/main/java/org/openapitools/codegen/plugin/CodeGenMojo.java

```diff
@@ -443,6 +443,12 @@ public class CodeGenMojo extends AbstractMojo {
     @Parameter(name = "generateModels", property = "openapi.generator.maven.plugin.generateModels")
     private Boolean generateModels = true;
 
+    /**
+     * Generate the models recursively if models should generate selectively (see modelsToGenerate) and all dependent models are to generate
+     */
+    @Parameter(name = "generateRecursiveDependentModels", property = "openapi.generator.maven.plugin.generateRecursiveDependentModels")
+    private Boolean generateRecursiveDependentModels = false;
+
```

Comment:
personally i suggest we start without another option and simply enable this in the `generateModels` feature.

Reply:
That would change the behavior of the options a lot. This would possibly break the processes of the users of the project. Because suddenly much more would be generated than expected. Backwards compatibility was the driving force behind the option. If removing the option is still desired, I can do so and thereby break backwards compatibility.

Reply:
if this result in additionl files not wanted by the users, they can use openapi gernator ignore list to skip those models

Reply:
now I have a huge list of models to generate. Without the switch another one will have a huge list to skip models. I mean to have a posibility to give some list to generate or skip is good. But the change behavior of generating is another point, I think.